### PR TITLE
Edited the static elements in context of CC

### PR DESCRIPTION
**Pull request by Bhavy0906**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -146,7 +147,7 @@ <h2>Two or two hundred. If you’ve got something to tell Ravi <span>before he l
 <img class="ravi_pic class_disable" src="img/ravi7.png" alt="ravi">
 </div>
 <footer>
-Made with<img class="heart animated animate_ultd flash class_disable" src="img/heart.png" alt="heart">at <a href="http://privateunlimited.com/" target="_blank">Private Unlimited</a>
+Made with<img class="heart animated animate_ultd flash class_disable" src="img/heart.png" alt="heart">by Coding Club of BPPC</a>
```
Comment (dmahajan980) [inline index.html:150]: Please change the last few words to `Coding Club, BITS Pilani` instead.

File: index.html [hunk 2]
```diff
@@ -46,9 +46,10 @@
 <div class="div_carlo">
 <img class="carlo_mob class_disable" src="img/carlo.gif" alt="image">
 <div class="carlo_txt animated" style="display:none;">
-<h2>Some 10 years ago, on a sultry April morning, a bright young man found himself standing outside a pretty purple villa at Dona Paula. </h2>
-<p>It was Ravi Shankar’s first day at Monte Carlo, home to Synapse and its many PGs (Project Groups, not Paying Guests). <br><br>
-Within a few short months of joining Synapse, he proved himself worthy of being elevated to a Business Head. For the better part of the next 10 years, Ravi has diligently led Synapse from the front. Today is the day we celebrate his journey.</p>
+<h2>Some 4/5 years ago, on a sweltering day of the month of August, a bright young engineer stood overwhelmed by grandeur of the Clock Tower. </h2>
```
Comment (dmahajan980) [inline index.html:49]: Also, replace this with "Some 4/5 years ago, on a sweltering day of August, a bright young engineer stood overwhelmed by the grandeur of the Clock Tower."

**Comment by dmahajan980** [comment]
@Bhavy0906 with the last few changes, this should be good to go.

**Review by TejasTiwari — APPROVED** [review]
Looks good

**Comment by dmahajan980** [comment]
This has been covered in #3, so we can close it safely.

**Pull request closed without merging**